topic_cleaning/aliases.json

```diff
@@ -0,0 +1,51 @@
+{
+	"ai": "artificial-intelligence",
+	"analytics": "data-analysis",
+	"ananlyis": "data-analysis",
+	"chatterbot": "chat-bot",
+	"deeplearning": "deep-learning",
+	"gan": "generative-adversarial-network",
+	"machine-learning-algorithms": "machine-learning",
+	"ml-learn": "machine-learning",
+	"ml": "machine-learning",
+	"neural nets": "neural-networks",
+	"neural-network": "neural-networks",
+	"nlp" : "natural-language-processing",
+	"nn": "neural-networks",
+	"secure": "security",
+	"stock-market": "stock-market-prediction",
+	"stock-prediction": "stock-market-prediction",
+	"tf" : "tensorflow",
+	"topic-modelling": "topic-modeling",
+	"console": "terminal",
+	"cli": "command-line",
+	"decorator": "decorators",
+	"lint": "linter",
+	"restful": "REST",
+	"RESTapi": "REST",
+	"dataframe": "data-frame",
+	"dataframes": "data-frame",
+	"data-science": "data-analysis",
+  	"newsfeed" : "news-feed",
+  	"webdriver" : "web-driver",
+  	"webserver": "web-server",
+  	"amazon": "aws",
+  	"json-serializer": "json-serialization",
+  	"hashmap": "hash-map",
+  	"crypto": "cryptography",
+  	"operations": "operation",
+  	"datetime": "date-time",
+  	"ip-address": "ip",
+  	"visualize": "visualization",
+  	"visualise": "visualization",
```

Comment:
@jpopelka that is saved on purpose. I don't know how to let the the system understand that with different spelling, they still mean the same. So for some time, these duplicates will be present.